### PR TITLE
app logo added

### PR DESCRIPTION
**Pull request by pranjalee30**
Hey @anisharaz App logo added instead of favicon.ico in the codebase[Issue #4 ]( https://github.com/blockx3/imp/issues/4#issue-2569324614)
Kindly review it and merge it

### CODE REVIEW

File: apps/web/app/IMPLOGO.png
Comment (anisharaz) [inline apps/web/app/IMPLOGO.png:1]: Commiting this large image file along with code is not a good idea , I will upload this to a CDN. Please remove this from commit

File: apps/web/app/favicon.ico
Comment (anisharaz) [inline apps/web/app/favicon.ico:1]: convert the logo into a fevicon.ico file and just replace the fevicon.ico file instead of deleting the file.

File: apps/web/middleware.ts
Comment (anisharaz) [inline apps/web/middleware.ts:1]: no changes in the middleware.ts fis is required please undo the changes.

**Review by anisharaz — CHANGES_REQUESTED** [review]
@pranjalee30 The logo is good , please make the required changes and it will be ready to merge.

**Comment by pranjalee30** [comment]
Hey @anisharaz  Once check please, I have added favicon as per your requirements
